src/order/galois_connection.lean

```diff
@@ -82,53 +82,59 @@ lemma l_u_le (a) : l (u a) ≤ a :=
 gc.l_le $ le_refl _
 
 lemma monotone_u : monotone u :=
-assume a b H, gc.le_u (le_trans (gc.l_u_le a) H)
+λ a b H, gc.le_u ((gc.l_u_le a).trans H)
 
 lemma monotone_l : monotone l :=
-assume a b H, gc.l_le (le_trans H (gc.le_u_l b))
+λ a b H, gc.l_le (H.trans (gc.le_u_l b))
 
 lemma upper_bounds_l_image_subset {s : set α} : upper_bounds (l '' s) ⊆ u ⁻¹' upper_bounds s :=
-assume b hb c, assume : c ∈ s, gc.le_u (hb (mem_image_of_mem _ ‹c ∈ s›))
+λ b hb c hc, gc.le_u (hb (mem_image_of_mem _ hc))
 
 lemma lower_bounds_u_image_subset {s : set β} : lower_bounds (u '' s) ⊆ l ⁻¹' lower_bounds s :=
-assume a ha c, assume : c ∈ s, gc.l_le (ha (mem_image_of_mem _ ‹c ∈ s›))
+λ a ha c hc, gc.l_le (ha (mem_image_of_mem _ hc))
 
 lemma is_lub_l_image {s : set α} {a : α} (h : is_lub s a) : is_lub (l '' s) (l a) :=
 ⟨gc.monotone_l.mem_upper_bounds_image $ and.elim_left ‹is_lub s a›,
-  assume b hb, gc.l_le $ and.elim_right ‹is_lub s a› $ gc.upper_bounds_l_image_subset hb⟩
+  λ b hb, gc.l_le $ and.elim_right ‹is_lub s a› $ gc.upper_bounds_l_image_subset hb⟩
 
 lemma is_glb_u_image {s : set β} {b : β} (h : is_glb s b) : is_glb (u '' s) (u b) :=
 ⟨gc.monotone_u.mem_lower_bounds_image $ and.elim_left ‹is_glb s b›,
-  assume a ha, gc.le_u $ and.elim_right ‹is_glb s b› $ gc.lower_bounds_u_image_subset ha⟩
+  λ a ha, gc.le_u $ and.elim_right ‹is_glb s b› $ gc.lower_bounds_u_image_subset ha⟩
 
 lemma is_glb_l {a : α} : is_glb { b | a ≤ u b } (l a) :=
-⟨assume b, gc.l_le, assume b h, h $ gc.le_u_l _⟩
+⟨λ b, gc.l_le, λ b h, h $ gc.le_u_l _⟩
 
 lemma is_lub_u {b : β} : is_lub { a | l a ≤ b } (u b) :=
-⟨assume b, gc.le_u, assume b h, h $ gc.l_u_le _⟩
+⟨λ b, gc.le_u, λ b h, h $ gc.l_u_le _⟩
 
 end
 
 section partial_order
```

Comment:
is it maybe worth combining both `partial_order` sections and just specifiying the parameters in the signature? it looks quite clunky (but maybe only in the diff view, I read it in the normal view and it seemed better)

Reply:
Yeah, I don't know. I only moved one lemma above two others. But also each section is only used for two lemmas.